linux-exploit-suggester.sh

```diff
@@ -1410,6 +1410,16 @@ author: blasty
 EOF
 )
 
+EXPLOITS_USERSPACE[((n++))]=$(cat <<EOF
+Name: ${txtgrn}[CVE-2017-5618]${txtrst} setuid screen v4.5.0 LPE
+Reqs: pkg=screen,ver>=4.5.0,ver<=4.5.0
+Tags: 
+Rank: 1
+analysis-url: https://seclists.org/oss-sec/2017/q1/18
```

Comment:
It looks like the final part of the URL was truncated `18` instead of `181`.

* https://seclists.org/oss-sec/2017/q1/181

However, there's no analysis at that URL. Here's a better one:

```suggestion
analysis-url: https://lists.gnu.org/archive/html/screen-devel/2017-01/msg00025.html
```

Reply:
Fine for me :)

Reply:
> 
> 
> Fine for me :)

No it isn't. It is clearly the wrong URL.

You meant to use this:

* https://seclists.org/oss-sec/2017/q1/181

Not this:

* https://seclists.org/oss-sec/2017/q1/18

Also, the URL you meant to use is the request for a CVE. That's not particularly useful - why not link directly to the analysis?

Reply:
Yes, originally I meant https://seclists.org/oss-sec/2017/q1/184 because this does include more information than the screen-devel ML one, like in which commit the issue was introduced.